packages/common-frontend/src/features/ide/slice.ts

```diff
@@ -31,6 +31,7 @@ type InitialState = {
   showBacklinks?: boolean;
   showOutwardLinks?: boolean;
   showHierarchy?: boolean;
+  isLocked?: boolean;
```

Comment:
We could also convert the state into local-state inside `DendronApp` as long we only need it there. Maybe premature to put it already inside the global state.

Reply:
fine for now. we can simplify later if needed